feedsx/src/main/java/com/likeminds/feedsx/post/detail/util/PostDetailUtil.kt

```diff
@@ -121,7 +122,8 @@ object PostDetailUtil {
                     return@setOnClickListener
                 }
                 // creates a route and returns an intent to handle the link
-                val intent = Route.handleDeepLink(context, url)
+                val updatedUrl = url.getUrlIfExist()
+                val intent = Route.handleDeepLink(context, updatedUrl)
```

Comment:
Have you tested, others links with change?

Reply:
Yes I have tested and it is working perfectly fine with all type of links.